src/bigdecimal.ts

```diff
@@ -2956,6 +2967,101 @@ export class BigDecimal {
         }
     }
 
+    /**
+    * Alias for `compareTo(val) === 0`.
+    * Consider using {@link equals} in case the precision needs to be considered.
+    * @returns true if the value is the same as `val`
+    * @see {@link equals}
+    * @see     {@link compareTo}
```

Comment:
```suggestion
    * @see {@link compareTo}
```

Reply:
Fixed in b9e0c777a81f9fc9283be03ed134960ef9dee293